src/supla/network/esp32eth.h

```diff
@@ -0,0 +1,208 @@
+/*
+  Copyright (C) AC SOFTWARE SP. Z O.O.
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+*/
+
+#ifndef esp_eth_h__
+#define esp_eth_h__
+
+#include <Arduino.h>
+
+#include <ETH.h>
```

Comment:
Please add link or name to library that has to be installed here

Reply:
there is nothing to install, it is part of arduino-esp32.
[https://github.com/espressif/arduino-esp32/tree/1.0.6/libraries/WiFi/src](url)